CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ Please respect the following instructions:
 
 # Changelog
 
+## 2024-01-01
+
+- [Improvement] Happy New Year!
+    Fix compatibility issue with Python 3.12 by removing dependency on `pkg_resources`.
+    Cookiecutter hooks to check input data validation.
+    Various licenses support.
+    New documentation format.
+    GitHub Actions for new plugins.
+    `dev` mode added to extra entries.
+  (by @CodeWithEmad).
```

Comment:
```suggestion
- [Improvement] Happy New Year!
  - Fix compatibility issue with Python 3.12 by removing dependency on `pkg_resources`.
  - Cookiecutter hooks to check input data validation.
  - Various licenses support.
  - New documentation format.
  - GitHub Actions for new plugins.
  - `dev` mode added to extra entries.
  (by @CodeWithEmad).
```
Without the bullet points, this will all render on one line.